service/src/main/java/db/postgres/V2_0_1__InitializeIdentityZones.java

```diff
@@ -53,8 +54,8 @@ public void migrate(final JdbcTemplate jdbcTemplate) throws Exception {
     }
 
     private Long createDefaultAuthzZone(final JdbcTemplate jdbcTemplate) {
-        final String insertZoneSql = "INSERT INTO authorization_zone (name, description, subdomain) "
-                + "VALUES (?,?,?)";
+        final String insertZoneSql =
```

Comment:
Yes I was worried about this as well. I remember that flyway has a checksum field in the schema table. Should I try to deploy this to RC to see if the migration goes OK before merging to develop?